lighthouse/classes/events/biosero/source_no_pickable_samples.py

```diff
@@ -50,3 +51,8 @@ def _create_message(self) -> Any:
             self.properties[property_name].add_to_warehouse_message(message)
 
         return message.render()
+
+    def process_event(self) -> None:
+        super().process_event()
+
+        self.transfer_to_bin()
```

Comment:
For instance, don't you need to check here too?

Reply:
Done